server/controllers/article.controller.js

```diff
@@ -204,5 +204,26 @@ class ArticleController {
     await sendMail(mailOption);
     return Response(res, 200, 'Article shared successfully');
   }
+
+  /**
+   * Delete article
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} response
+   */
+  static async deleteArticle(req, res) {
+    const {
+      params: { slug }, user: { id }
+    } = req;
+
+    const article = await Article.destroy({
```

Comment:
@Chiazokam for articles that include images, does this endpoint delete article images on cloudinary? if it does not, It is necessary to delete images alongside the articles. if you need help with that reach out to me.

Reply:
Did not take care of that. Thanks for drawing my attention

Reply:
Done